test/integration/cartridge_health_test.lua

```diff
@@ -38,3 +38,59 @@ g.test_cartridge_health_fail_handler = function()
         box.info = _G.old_info
     ]])
 end
+
+g.test_cartridge_health_handler_state_1 = function()
```

Comment:
please rename test to smth more understandable

Reply:
The same for the other test

Reply:
Sure. Thank you.